lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -9,7 +9,8 @@ class Base
           ::Errno::ENOENT,
           ::Racc::ParseError,
           ::RubyParser::SyntaxError,
-        ]
+          ::RuntimeError,
+        ].freeze
```

Comment:
Curious about what errors - if any - we'd want to make sure aren't rescuable? And why this engine seems to have a bit more error catching than others?